Changes

```diff
@@ -114,6 +140,10 @@ API
 - GraphComponent : Added protected `parentChanged()` virtual method (#3080).
 - ImagePlug : Added convenience methods for evaluating global image properties (#3073).
 - GraphComponentPath : Added property for accessing the GraphComponent (#3106).
+- LightFilter : Added LightFilter class used as base for renderer spcific implementations (#3020).
+- TaskMutex : Added TaskMutex as part of the foundation of a revamped caching scheme (#3133).
```

Comment:
Let's remove the mention of TaskMutex and LRUCache since they're private. Once my final PR is merged, we'll document the API-visible changes and add some mention of the resulting performance improvements.